integration/common.go

```diff
@@ -36,13 +36,16 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// nolint: megacheck
```

Comment:
per comment above, need to add the build tags for this folder to get megacheck to work correctly